GliaWidgets/Sources/View/Chat/Message/GvaPersistentButtonOptionView.swift

```diff
@@ -0,0 +1,69 @@
+import UIKit
+
+class GvaPersistentButtonOptionView: UIView {
```

Comment:
Please use `BaseView`  and override `setup` + `defineLayout` methods   
```suggestion
class GvaPersistentButtonOptionView: BaseView {
```

Reply:
fixed [6bf7cd5](https://github.com/salemove/ios-sdk-widgets/pull/683/commits/6bf7cd5b4a195ff9f90c3d6ed2b0b02b7d8ee1ec)